packages/application-shell/src/components/navbar/use-navbar-state-manager.ts

```diff
@@ -119,6 +119,47 @@ const useNavbarStateManager = (props: HookProps) => {
           })
           .filter(nonNullable)
       : [];
+  const organizationCustomAppsMenu =
+    projectExtensionsQuery &&
+    projectExtensionsQuery.projectExtension &&
+    projectExtensionsQuery.projectExtension.installedApplications
+      ? projectExtensionsQuery.projectExtension.installedApplications
+          .map<TNavbarMenu | undefined>((installedApplication) => {
+            const application = installedApplication.application;
+            // Map the menu properties to match the one from the proxy schema.
+            // This is to ensure that the menu object is the same from the proxy
+            // config and the custom apps config, thus allowing them to be
+            // concatenated and rendered the same way.
+            if (!application.menuLinks) return;
```

Comment:
I don't think we need this 🤔

Reply:
I think we do, since it is a `Maybe`

https://github.com/commercetools/merchant-center-application-kit/blob/43aace3e1bc247183f7c6931d68c405f0878a8e5/packages/application-shell/src/types/generated/settings.ts#L306:L306